pulp_ansible/app/viewsets.py

```diff
@@ -368,6 +377,12 @@ class AnsibleNamespaceFilter(ContentFilter):
     A filter for namespaces.
     """
 
+    # this is a hack to keep pulp from throwing a 400 when an unknwon
+    # query param is included, since filtering is handled by the related
+    # field serializer
+    include_related = filters.CharFilter(method="no_op")
+    my_permissions = filters.CharFilter(method="has_permissions")
+
```

Comment:
This part I don't get. These are pulp api views.
Also we don't break down permissions to the content. I believe we should have the `RolesMixin` in the `AnsibleGlobalNamespaceViewSet` instead.

Reply:
Those fields must be declared, otherwise we are not able to pass those fields as query params.